indra/newview/llheroprobemanager.cpp

```diff
@@ -194,11 +195,17 @@ void LLHeroProbeManager::update()
         else
         {
             mNearestHero = nullptr;
+            mDefaultProbe->mViewerObject = nullptr;
         }
 
         mHeroProbeStrength = 1;
     }
+    else
+    {
+        mNearestHero = nullptr;
+        mDefaultProbe->mViewerObject = nullptr;
```

Comment:
I think this `LLReflectionMap::mViewerObject` may also need to be made into an LLPointer<> to be safe for these kind of problems as well